packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx

```diff
@@ -29,28 +29,25 @@ export const BridgeTxItem = ({
 }: BridgeTxItemProps) => {
   return (
     <CardList.Item onClick={onClick} css={styles.listItem}>
-      <Box.Flex align="center" css={styles.wrapper}>
+      <Box.Flex css={styles.wrapper}>
```

Comment:
lets do like this one

<img width="700" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/b2eaff55-8ac1-46ae-b880-3251f03e221a">

- timestamp left aligned
- other items right aligned (direction, asset / amount, status)
- could experiment putting some labels to identify, like this one:


<img width="654" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/5e14b1c9-11a7-4183-8c7d-106c06ba3bbf">